tests/test_port_dpb_system.py

```diff
@@ -248,25 +242,380 @@ def test_port_breakout_with_acl(self, dvs):
 
         # Verify port is removed from ACL table
         self.dvs_acl.verify_acl_table_count(1)
-
-        #TBD: Uncomment this, or explain why Ethernet0 is being added back to ACL table
-        # Also, string "None" is being added as port to ACL port list after the breakout
-        #self.dvs_acl.verify_acl_group_num(0)
+        self.dvs_acl.verify_acl_group_num(0)
 
         # Verify child ports are created.
         self.verify_only_ports_exist(dvs, ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3"])
 
-        # Enable below snippet after fixing the above issues
-        '''
         # Move back to 1x100G[40G] mode and verify current mode
         dvs.change_port_breakout_mode("Ethernet0", "1x100G[40G]", "-f")
         dpb.verify_port_breakout_mode(dvs, "Ethernet0", "1x100G[40G]")
-        '''
 
         # Remove ACL table and verify the same
         self.dvs_acl.remove_acl_table("test")
         self.dvs_acl.verify_acl_table_count(0)
 
+    def test_cli_command_with_hyphen_f_option(self, dvs):
+
+        dvs.setup_db()
+        dpb = DPB()
+        self.setup_db(dvs);
+
+        portGroup = ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3"]
+        rootPortName = portGroup[0]
+        vlanID = "100"
+        aclTableName = "DPB_ACL_TBL_1"
+        breakoutMode1x = "1x100G[40G]"
+        breakoutMode2x = "2x50G"
+        breakoutMode4x = "4x25G[10G]"
+        breakoutOption = "-f" #Force breakout by deleting dependencies
+
+        # Breakout port with no dependency using "-f" option
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode4x, breakoutOption)
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode1x, breakoutOption)
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode1x)
+
+        # Breakout port with VLAN and ACL dependency
+
+        # Create ACL table and bind port
+        self.dvs_acl.verify_acl_group_num(0)
+        bind_ports = []
+        bind_ports.append(rootPortName)
+        self.dvs_acl.create_acl_table(aclTableName, "L3", bind_ports)
+        self.dvs_acl.verify_acl_group_num(1)
```

Comment:
Why this has single group number? is it because for h/w it is just one port?. I thought it will create 4 groups.